planetarypraxis/smartforests
# feat: add endnotes field

## Description
Adds an endnotes field for stories and radio episodes, displayed below the main content.

## How Can It Be Tested?
Edit a story page, add content to the endnotes field, and check it is displayed on the published page.

## How Will This Be Deployed?
Migration required.

## Types of changes


## Checklist:


- * I've checked the spec (e.g. Figma file) and documented any divergences.
- * My change requires a change to the documentation.
- * I've updated the documentation accordingly.
- * Replace unused checkboxes with bullet points.
